bin/startPipeline.sh

```diff
@@ -98,7 +98,7 @@ function generateScripts () {
 		log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "${_controlFileBase}.finished nor ${_controlFileBase}.started exists."
 		log4Bash 'INFO'  "${LINENO}" "${FUNCNAME:-main}" '0' "Generating scripts for ${_project} ..."
 	fi
-	
```

Comment:
Please leave blank lines indented: required for block indentation changes by some editors.